tools/sof-dump-status.py

```diff
@@ -344,6 +345,11 @@ def _covertType(type_lst):
         print(card_info['longname'])
         exit(0)
 
+    if ret_args.get('dsp_status') is True:
+        sysinfo.loadPower()
+        print(sysinfo.sys_power['run_status'][0]['status'])
```

Comment:
how about `print(sysinfo.sys_power['run_status'][ret_args['S']]['status'])`
```
parser.add_argument('-S', '--dsp_status', type=int, help='get target id dsp power status')
```

Reply:
sysinfo.sys_power['run_status'] is a list, and I think it always contains one element, is this true?, so i use index 0.
change argument type to int is error prone, because the user never knows which int number should given.

Reply:
you can load from case like by this `$SOFCARD`  
because we already meet some patch from kernel change this value from 0 to 1  
you can catch information from [config.sh](https://github.com/thesofproject/sof-test/blob/master/case-lib/config.sh#L38) and [lib.sh](https://github.com/thesofproject/sof-test/blob/master/case-lib/lib.sh)
```
# setup SOFCARD id
if [ ! "$SOFCARD" ];then
    SOFCARD=$(grep '\]: sof-[a-z]' /proc/asound/cards|awk '{print $1;}')
fi
```
On the other hand, we have more than one sound card, so ...

Reply:
modified with SOFCARD,